src/rehype-tabs/plugins.js

```diff
@@ -0,0 +1,139 @@
+import { visit } from 'unist-util-visit'
```

Comment:
Can this be a TypeScript instead of JS file?

Reply:
Also same comment as the `initializeTabs.ts` file, this could ideally be turned into an Astro component.